website/projects/tests/tests_aws/test_awssync_refactored.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for awssync_refactored.py."""
+import json
+from unittest.mock import patch
+
+from botocore.exceptions import ClientError
+
+from django.test import TestCase
+
+from moto import mock_organizations
+
+from courses.models import Semester
+
+from projects.aws.awssync_refactored import AWSSyncRefactored
+from projects.aws.awssync_structs import AWSTree, Iteration, SyncData
+
+
+@mock_organizations
+class AWSSyncRefactoredTest(TestCase):
+    def setUp(self):
+        self.sync = AWSSyncRefactored()
+
+    def test_create_course_ou__not_exists(self):
+        self.sync.api_talker.create_organization(feature_set="ALL")
+        root_id = self.sync.api_talker.list_roots()[0]["Id"]
+        tree = AWSTree("root", root_id, [])
+
+        with patch.object(Semester.objects, "get_or_create_current_semester", return_value="Spring 2023"):
```

Comment:
I believe this line is unnecessary because the current semester does not affect the rest of the test (the test does nothing with the OU name), so I would suggest removing this  line or also asserting that the OU name is "Spring 2023".